### DISCUSSION TITLE
Forslag til utvidelser av kurset

### DISCUSSION THREAD

**Discussion by thomaswolff** [nrkno/dotnetskolen#4] [Ideas]
Veiledning
- Oppdatér README.md med egne kodesnutter for det man skal lime inn i terminalen. Da blir det enklere å kopiere fra GitHub.

Ekstraoppgaver
- Steg 13 - logging

**Comment by thomaswolff** [comment]
Basert på #5 kan vi vurdere å lage et parallelt spor for kurset med C#

**Comment by matiasp** [comment]
Deploy til f.eks. AKS

**Reply by thomaswolff** [reply]
_Insert link to Docker-skolen og Kubernetes-skolen_ 😄

**Reply by thomaswolff** [reply]
⬆️ Svaret over var litt flåsete. Deploy er et godt innspill, og det er en naturlig fortsettelse når man har en app som fungerer. Jeg tenker at det kunne vært et fint scope for Docker-skolen v.01, og at .NET-skolen sin rolle er hvordan publisere noe som kan deploye. Så kanskje utvide med info om `dotnet publish`?

**Reply by matiasp** [reply]
Hehe, skjønner hva du mener! Ja, info om `dotnet publish` duger nok. Tenkte bare at når man sitter med en kul, fungerende applikasjon, så hadde det vært interessant å se hvordan man kan deploye den og få vist den til andre også 😃

**Reply by thomaswolff** [reply]
Ja, helt enig 🙂 En måte å gjøre det på er å utvide .NET-skolen med en enkel beskrivelse av hvordan deploye til en gitt plattform (Kubernetes, Azure App Services, AWS, Google Cloud etc.), og så gå dypere inn i materien i en separat workshop.
